doc-surrealdb_versioned_docs/version-1.x/surrealql/statements/info.mdx

```diff
@@ -14,30 +14,46 @@ INFO FOR [
 	ROOT
 	| NS | NAMESPACE
 	| DB | DATABASE
-	| SCOPE @scope
 	| TABLE @table
+	| USER @user [ON @level]
+	| SCOPE @scope
 ];
 ```
 
+The information returned from an `INFO` command is an object containing items that almost always correspond to a matching [DEFINE](/docs/surrealdb/surrealql/statements/define) statement. For example, the `INFO FOR NS` command returns the information on the databases, tokens, and users of a namespace, which are defined with `DEFINE DATABASE`, `DEFINE TOKEN`, and `DEFINE USER` statements.
+
 ## Example usage
 
 There are a number of different `INFO` commands for retrieving the configuration at the different levels of the database.
 
-### System information
+### System (root) information
```

Comment:
System users are (Root, namespace and database) can make this a H2 and make the rest H3
```suggestion
## System  information
```